components/molecule/photoUploader/src/index.scss

```diff
@@ -0,0 +1,326 @@
+$c-photo-uploader-action-buttons-hover: $c-white !default;
+$c-photo-uploader-drag-photo-rejected: $c-white !default;
+$c-photo-uploader-skeleton: $c-primary !default;
+$c-photo-uploader-skeleton-hover: $c-white !default;
+$c-photo-uploader-initial-state: $c-primary !default;
+$jc-photo-uploader: center !default;
+$p-photo-uploader: $p-m !default;
+$p-photo-uploader-desktop: $p-xl !default;
+$w-photo-uploader-thumb-image: 100% !default;
+$w-photo-uploader-thumb-image-desktop: 160px !default;
+$s-photo-uploader-thumb-icons: 16px !default;
+$fz-photo-uploader-initial-state: $fz-s !default;
+
+$mih-photo-uploader: 249px;
+$s-photo-uploader-counter: 16px;
+$h-photo-uploader-thumb-image: ($w-photo-uploader-thumb-image * 3) / 4;
+$h-photo-uploader-thumb-image-desktop: (
+    $w-photo-uploader-thumb-image-desktop * 3
+  ) / 4;
+$w-photo-uploader-skeleton: $w-photo-uploader-thumb-image;
+$w-photo-uploader-skeleton-desktop: $w-photo-uploader-thumb-image-desktop;
+$maw-photo-uploader-thumb-drag-text: 200px;
+
+#{$base-class} {
+  &-dropzone {
+    background: $bg-photo-uploader;
+    border-radius: $bdrs-l;
+    color: $c-photo-uploader;
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    min-height: $mih-photo-uploader;
+    outline: none;
+    padding: $p-photo-uploader;
+    position: relative;
+
+    @include media-breakpoint-up(m) {
+      padding: $p-photo-uploader-desktop;
+    }
+
+    &--disabled {
+      cursor: no-drop;
+    }
+  }
+  &-initialState {
+    align-items: center;
+    border: $bd-photo-uploader;
+    border-radius: $bdrs-l;
+    color: $c-photo-uploader-initial-state;
+    display: flex;
+    flex-direction: column;
+    font-size: $fz-photo-uploader-initial-state;
+    height: 100%;
+    justify-content: center;
+    left: 0;
+    line-height: $lh-m;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  &-textState {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+
+    &Text {
+      margin-top: $m-m;
+    }
+    &Divider {
+      margin: $m-m 0;
+    }
+  }
+
+  &-dragState {
+    align-items: center;
+    border-radius: $bdrs-l;
+    color: $c-photo-uploader-drag-photo;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 100%;
+    left: 0;
+    min-height: $mih-photo-uploader;
+    position: absolute;
+    top: 0;
+    width: 100%;
+
+    &-textState {
+      line-height: $lh-m;
+      max-width: $maw-photo-uploader-thumb-drag-text;
+      text-align: center;
+    }
+  }
+
+  &-dragState--accepted {
+    background-color: $bg-photo-uploader-drag-photo;
+    border: $bd-photo-uploader;
+  }
+
+  &-dragState--rejected {
+    background-color: $bg-photo-uploader-drag-photo-rejected;
+    border: $bd-photo-uploader-drag-photo-rejected;
+    color: $c-photo-uploader-drag-photo-rejected;
+    z-index: $z-notifications;
+  }
+
+  &-preview-fallback {
+    border-radius: $bdrs-rounded;
+  }
+
+  &-preview {
+    display: grid;
```

Comment:
I love it! 🤩
We don't need support *IE 11*?

Reply:
yes...we need... :(

Reply:
We're gonna check it ASAP! But there are some problems right now wit `sui-studio` and `IE11`, so we can't test it properly 🙄